data/i18n/zh-cn/zh-cn.toml

```diff
@@ -332,7 +332,7 @@ other = "订阅"
 other = "简介"
 
 [thirdparty_message]
-other = """本部分链接到提供 Kubernetes 所需功能的第三方项目。Kubernetes 项目作者不负责这些项目。此页面遵循<a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF 网站指南</a>，按字母顺序列出项目。要将项目添加到此列表中，请在提交更改之前阅读<a href="/docs/contribute/style/content-guide/#third-party-content">内容指南</a>。"""
+other = """本部分链接到提供 Kubernetes 所需功能的第三方项目。Kubernetes 项目作者不负责这些项目。此页面遵循<a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF 网站指南</a>，按字母顺序列出项目。要将项目添加到此列表中，请在提交更改之前阅读<a href="/zh-cn/docs/contribute/style/content-guide/#third-party-content">内容指南</a>。"""
```

Comment:
Looks like this doesn't work.

Reply:
According to the deployment link, It can now jump to the Chinese document normally.
<img width="874" alt="image" src="https://github.com/kubernetes/website/assets/20641704/5dbab713-0b3c-4bec-b0bd-0075dca1d9b2">
https://deploy-preview-43268--kubernetes-io-main-staging.netlify.app/zh-cn/docs/contribute/style/content-guide/#third-party-content

Reply:
I checked the netlify preview log, it complained on this line. Then I triggered a rebuild, looks like it works now.